Assets/HoloToolkit/Input/Scripts/InputSources/GesturesInput.cs

```diff
@@ -3,7 +3,7 @@
 
 using UnityEngine;
 
-#if UNITY_EDITOR || UNITY_WSA
```

Comment:
Really it shouldn't matter even when remoting because the build should already be WSA